tests/foreman/cli/test_discoveryrule.py

```diff
@@ -172,15 +175,32 @@ def test_positive_create_with_org_loc_id(
         assert class_org.name in rule.organizations
         assert class_location.name in rule.locations
 
+        new_org = Box(make_org())
+        new_loc = Box(make_location())
+        new_hostgroup = Box(
+            make_hostgroup({'organization-ids': new_org.id, 'location-ids': new_loc.id})
+        )
+        target_sat.cli.DiscoveryRule.update(
+            {
+                'id': rule.id,
+                'organization-ids': new_org.id,
+                'location-ids': new_loc.id,
+                'hostgroup-id': new_hostgroup.id,
+            }
+        )
+        rule = Box(DiscoveryRule.info({'id': rule.id}))
```

Comment:
same for other occurences
```suggestion
        rule = Box(target_sat.cli.DiscoveryRule.info({'id': rule.id}))
```

Reply:
Tried to cover every occurence